stone/target/python_rsrc/stone_base.py

```diff
@@ -34,6 +34,16 @@ def __init__(self, tag, value=None):
         self._tag = tag
         self._value = value
 
+    def __eq__(self, other):
```

Comment:
If someone passes in an instance of `object()`, this function will fail with a runtime error, right?  I'm not sure what the contract of `__eq__` is, but maybe you're required to return `false` in that case?